src/Onboarding/Processor/BasicOnboardingProcessor.php

```diff
@@ -17,23 +18,27 @@ final class BasicOnboardingProcessor implements OnboardingProcessorInterface
 {
     private ClientInterface $httpClient;
 
+    private RequestFactoryInterface $requestFactory;
+
     private SellerWebhookRegistrarInterface $sellerWebhookRegistrar;
 
     private string $url;
 
     public function __construct(
         ClientInterface $httpClient,
+        RequestFactoryInterface $requestFactory,
```

Comment:
I'm afraid that introducing a new service as both not nullable and not the last one would result in a BC break.

Reply:
I've removed BC